tests/acceptance/standard/test_project_avatar.py

```diff
@@ -0,0 +1,292 @@
+
+        # Refresh project data
+        project = project.manager.get(project.id)
+
+        # Verify avatar is set
+        assert project.avatar_url is not None
+
+    def test__project_avatar_paths_comprehensive(self, project):
+        """Test both absolute and relative paths comprehensively"""
+
+        # Test 1: Absolute path (already tested above, but included for completeness)
+        config_absolute = f"""
+        projects_and_groups:
+          {project.path_with_namespace}:
+            project_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config_absolute, project)
+
+        project = project.manager.get(project.id)
+        assert project.avatar_url is not None
+
+        # Test 2: Simple relative path
+        original_cwd = os.getcwd()
+        temp_dir = tempfile.mkdtemp()
+
+        try:
+            os.chdir(temp_dir)
+
+            # Copy image to temp directory
+            temp_image_name = "relative_test_avatar.png"
+            shutil.copy2(self.test_image_path, temp_image_name)
+
+            config_relative = f"""
+            projects_and_groups:
+              {project.path_with_namespace}:
+                project_settings:
+                  avatar: "{temp_image_name}"
+            """
+            run_gitlabform(config_relative, project)
+
+            project = project.manager.get(project.id)
+            assert project.avatar_url is not None
+
+        finally:
+            os.chdir(original_cwd)
+            shutil.rmtree(temp_dir)
+
+        # Test 3: Relative path with subdirectory
+        temp_dir2 = tempfile.mkdtemp()
+        try:
+            os.chdir(temp_dir2)
+
+            subdir = "test_images"
+            os.makedirs(subdir)
+            relative_subdir_path = os.path.join(subdir, "subdir_avatar.png")
+            shutil.copy2(self.test_image_path, relative_subdir_path)
+
+            config_relative_subdir = f"""
+            projects_and_groups:
+              {project.path_with_namespace}:
+                project_settings:
+                  avatar: "{relative_subdir_path}"
+            """
+            run_gitlabform(config_relative_subdir, project)
+
+            project = project.manager.get(project.id)
+            assert project.avatar_url is not None
+
+        finally:
+            os.chdir(original_cwd)
+            shutil.rmtree(temp_dir2)
+
+    def test__project_avatar_delete(self, project):
+        # First ensure there's an avatar to delete by setting one
+        config_set = f"""
+        projects_and_groups:
+          {project.path_with_namespace}:
+            project_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config_set, project)
+
+        # Refresh project data to get current state
+        project = project.manager.get(project.id)
+
+        # Verify that avatar exists
+        assert project.avatar_url is not None, "Avatar should exist after setting it"
```

Comment:
I think we'd expect the project avatar to be set by the previous test, right? Do really need to run through gitlabform again before we begin the test? Wouldn't the last assert be sufficient before the test begins?